packages/waku/src/lib/handlers/dev-worker-impl.ts

```diff
@@ -50,36 +44,21 @@ const handleRender = async (mesg: MessageReq & { type: 'render' }) => {
       searchParams: new URLSearchParams(rr.searchParamsString),
       method: rr.method,
       context: rr.context,
-      body: rr.stream,
+      body: rr.stream!,
```

Comment:
How can we assure that it's not undefined?

Reply:
To be honest, I thought about this, and I could not find a solution, it required so many types gymnastics which I then could not get it right. Let me know if you have an idea.

Reply:
Ah, yeah, it's optional. 204a34e should do for now. We can probably accept `undefined` so that we don't need such syntax.